pom.xml

```diff
@@ -182,10 +182,29 @@ along with this program.  If not, see <http://www.gnu.org/licenses />.
                     </exclusion>
                 </exclusions>
             </dependency>
+            <dependency>
+                <groupId>com.codeablereason.restcompress.provider</groupId>
+                <artifactId>rest-compress-lib</artifactId>
+                <version>0.6</version>
+            </dependency>
         </dependencies>
     </dependencyManagement>
     <dependencies>
         <dependency>
+            <groupId>com.codeablereason.restcompress.provider</groupId>
```

Comment:
This dependency should be in crud and metadata, not root pom because it will then be included as a dependency for auth and common modules.  And exclusions would be better to put in dependencyManagement.